frontend/src/mocks/handlers.ts

```diff
@@ -91,6 +91,8 @@ export const defaultMocks: Mocks = {
         },
         '/api/projects/:team_id/groups/': EMPTY_PAGINATED_RESPONSE,
         '/api/projects/:team_id/groups_types/': [],
+        '/api/environments/:team_id/groups/': EMPTY_PAGINATED_RESPONSE,
+        '/api/environments/:team_id/groups_types/': [],
```

Comment:
i can see these as errors in the network tab
i don't _know_ whether it's linked but doesn't hurt to add them